content/spin/v2/python-components.md

```diff
@@ -26,25 +26,16 @@ With <a href="https://www.python.org/" target="_blank">Python</a> being a very p
 
 > This guide assumes you are familiar with the Python programming language, but if you are just getting started, be sure to check out <a href="https://docs.python.org/3/" target="_blank">the official Python documentation</a> and comprehensive <a href="https://docs.python.org/3/reference/" target="_blank">language reference</a>.
 
-[**Want to go straight to the Spin SDK reference documentation?**  Find it here.](https://fermyon.github.io/spin-python-sdk/v1)
+[**Want to go straight to the Spin SDK reference documentation?**  Find it here.](https://fermyon.github.io/spin-python-sdk)
 
-**Please note:** There is a blog article [introducing the release of this Spin Python SDK](https://www.fermyon.com/blog/spin-python-sdk) if you are interested in some further reading; in addition to this technical documentation.
+## Componentize-py
 
-## Spin's Python Plugin
-
-To compile Python programs to Spin components, you need to install a Spin plugin called `py2wasm`. The following commands will ensure that you have the latest version of the plugin installed:
-
-<!-- @selectiveCpy -->
+The Python SDK is built using [`componentize-py`](https://github.com/bytecodealliance/componentize-py). It can be installed using the following command:
 
 ```bash
-# Fetch all of the latest Spin plugins from the spin-plugins repository
-$ spin plugin update
-# Install py2wasm plugin
-$ spin plugin install py2wasm
+pip3 install componentize-py==0.11.0
```

Comment:
Since it has not reached 1.0 yet, The minor releases do have breaking changes. The version for componentize-py is set in the `requirements.txt` file within the template. I have removed the specific version of installation from this particular location though